modules/platform_stm32f302xb/platform_stm32f302xb.h

```diff
@@ -2,7 +2,8 @@
 
 #include <stdint.h>
 
-#define STM32F302xB
+// NOTE: in the CMSIS (stm32f3xx.h), this is #define used for the STM32F302xB
+#define STM32F302xC
```

Comment:
Why is the STM32F302xC used for the STM32F302xB?

Reply:
If you look in _ChibiOS/os/common/ext/CMSIS/ST/STM32F3xx/stm32f3xx.h_, it explicitly states:

```
 /* #define STM32F302x8 */   /*!< STM32F302K6, STM32F302K8, STM32F302C6, STM32F302C8,
                                   STM32F302R6 and STM32F302R8 Devices */
  /* #define STM32F302xC */   /*!< STM32F302CB, STM32F302CC, STM32F302RB, STM32F302RC,
                                   STM32F302VB and STM32F302VC Devices */
  /* #define STM32F302xE */   /*!< STM32F302RE, STM32F302VE, STM32F302ZE, STM32F302RD,
                                   STM32F302VD and STM32F302ZD Devices */
 ```

The other MCUs we use with ChibiOS (or have used), i.e. STM32F302R8 and STM32F302RE have their own explicit #define